pvm/ts/src/engine/actions/baseAction.test.ts

```
@@ -90,14 +89,14 @@
 
             it("should throw error if player is not active", () => {
                 jest.spyOn(game, "currentPlayerId", "get").mockReturnValue("0x980b8D8A16f5891F41871d878a479d81Da52334c");
-                jest.spyOn(game as any, "getPlayerStatus").mockReturnValue(PlayerStatus.FOLDED);
+                jest.spyOn(game, "getPlayerStatus").mockReturnValue(PlayerStatus.FOLDED);
 
-                expect(() => action.verify(player)).toThrow("Only active player can check.");
+                expect(() => action.verify(player)).toThrow("Must be currently active player.");
             });
 
-            it("should allow verification for active player on their turn", () => {
+            it.only("should allow verification for active player on their turn", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)